free-1.md

```diff
@@ -10,34 +13,75 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Legal writing is both notoriously complex and requires a lot of context to interpret.
+The users of the language would probably be lawyers, judges, lawmakers or reporters who wanted
+to dig into the context, history and use of various laws, definitions, rulings and interpretations.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Laws, rulings, etc. already need to be documented, and need to be accessed in a document form.
+A DSL is appropriate in that it can also supersede that use case, because it contains both
+the document form and metadata that is eminently useful for people browsing and researching
+the documents.  Additionally, it is helpful in that it could require a minimum of modification
+from the original documents to become syntactically valid (if not semantically useful).
```

Comment:
So one could take existing documents and add references as they please to make it your language? Sounds cool